src/overlay/PeerManager.cpp

```diff
@@ -199,6 +204,9 @@ PeerManager::loadRandomPeers(PeerQuery const& query, size_t size)
     std::tm nextAttempt = VirtualClock::pointToTm(mApp.getClock().now());
     int maxNumFailures = query.mMaxNumFailures;
     int outbound = query.mOutbound;
+    int preferred = query.mPreferred
```

Comment:
the column is called `flags`, so we should be consistent with that (variable name `preferred` -> `flags`)